scripts/docs.sh

```diff
@@ -74,7 +74,7 @@ fi
 BRANCH=`git branch | grep \* | cut -d ' ' -f2`
 echo "Current Branch: ${BRANCH}"
```

Comment:
It also seems we can remove this, as we're just printing the branch, which should now always be master